packages/client.api/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@osdk/api": "workspace:^",
+    "@osdk/internal.foundry": "workspace:^",
```

Comment:
If these types are re-exported (implicitly or explicitly) then they cannot be a devDependency without also being a peer dependency (or just a regular dependency).

If we are using this package for more than types, then making it a dev dependency also means it gets inlined into this package by tsup so that might also not be desirable.